roles/kubernetes/node/templates/kubelet-container.j2

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
-/usr/bin/docker run --privileged --rm \
---net=host --pid=host --name=kubelet \
+/usr/bin/docker run --privileged \
+--net=host --pid=host --name=kubelet --restart=always \
```

Comment:
Using `--restart=always` will cause infinite restarts if kubelet is in a startup crash loop. We could instead set a max # of retry attempts with something like `--restart=on-failure:5` for up to 5 attempts.